rundeckapp/grails-app/domain/rundeck/CommandExec.groovy

```diff
@@ -203,10 +227,13 @@ public class CommandExec extends WorkflowStep  {
         if (data.plugins) {
             ce.pluginConfig = data.plugins
         }
-        return ce
     }
 
     public boolean isNodeStep(){
         return true;
     }
+
+    public Boolean getNodeStep(){
```

Comment:
Is this used anywhere? Looks like it always returns true?

Reply:
This method conforms to groovy's expected format for getters and setters so that an instance of this class can use the form `command.nodeStep`. Unfortunately the form `isBoolean()` for boolean getters isn't sufficient.